src/main/scala/uk/gov/nationalarchives/tdr/api/service/ClientFileMetadataService.scala

```diff
@@ -39,17 +44,10 @@ class ClientFileMetadataService(clientFileMetadataRepository: ClientFileMetadata
     })
   }
 
-  private def rowToOutput(row: ClientfilemetadataRow): ClientFileMetadata = {
-    ClientFileMetadata(
-      row.fileid,
-      row.originalpath,
-      row.checksum,
-      row.checksumtype,
-      row.lastmodified.getTime,
-      row.createddate.getTime,
-      row.filesize,
-      row.datetime.getTime,
-      row.clientfilemetadataid
-    )
+  def getClientFileMetadata(fileId: UUID): Future[ClientFileMetadata] = {
```

Comment:
What's the reason for moving these methods around? The commit message says "Add test to Consignment and FileMetadata spec and fix typo".

Reply:
Ah, yes, I switched them around just so that this file matches the order that they appear in the ClientFileMetadata Repository. This was supposed to be a different commit; I can undo this.